content-review/impl/vericite/src/main/java/org/sakaiproject/contentreview/vericite/client/model/ExternalContentUploadInfo.java

```diff
@@ -105,6 +108,24 @@ public void setFilePath(String filePath) {
     this.filePath = filePath;
   }
 
+  public ExternalContentUploadInfo headers(Object headers) {
+    this.headers = headers;
+    return this;
+  }
+
+   /**
+   * A map of required headers
+   * @return headers
+  **/
+  @ApiModelProperty(example = "null", value = "A map of required headers")
+  public Object getHeaders() {
+    return headers;
+  }
+
+  public void setHeaders(Object headers) {
```

Comment:
Is this really going to be a variable Object? It kind of looks like from the code that it's expecting it to just be able to be easily convertible to a String?